Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "MLJModelInterface"
+uuid = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
+authors = ["Thibaut Lienart <ltib@me.com>"]
+version = "0.1.0"
+
+[deps]
+MLJScientificTypes = "2e2323e0-db8b-457b-ae0d-bdfb3bc63afd"
+
```

Comment:
Shouldn't the [deps] be ScientificTypes ?

Reply:
that's the case right?